viewing_party/party.py

```diff
@@ -1,23 +1,132 @@
+import statistics
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
-
+# -----------------------------------------
 def create_movie(title, genre, rating):
-    pass
+    # create movie information
+    movie_info = {}
+    if title == None or genre == None or rating == None:
+        return None
+    movie_info['title'] = title
+    movie_info['genre'] = genre
+    movie_info['rating'] = rating
+    return movie_info 
+
+
+def add_to_watched(user_data, movie):
+    # add movie to what user has watched
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    # add movie to user's watchlist
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+def watch_movie(user_data, title):
+    # add a watched movie to watched
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watched"].append(movie)
+    # remove watched movie from watchlist
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+    return user_data
 
 # -----------------------------------------
 # ------------- WAVE 2 --------------------
 # -----------------------------------------
+def get_watched_avg_rating(user_data):
+    combined_rating = 0
+    if user_data["watched"] == []:

```

Comment:
you don't need to compare an empty list here. The pythonic way would be `if not user_data["watched"]`